playground/src/router.ts

```diff
@@ -6,20 +6,65 @@ const routes = [
     name: 'Home',
     component: () => import('./pages/index.vue'),
   },
+  {
+    path: '/basic',
+    name: 'Basic',
+    component: () => import('./pages/TheBasic.vue'),
+  },
+  {
+    path: '/groups',
+    name: 'Groups',
+    component: () => import('./pages/TheGroups.vue'),
+  },
+  {
+    path: '/responsiveness',
+    name: 'Responsiveness',
+    component: () => import('./pages/Responsiveness.vue'),
+  },
+  {
+    path: '/conditional',
+    name: 'Conditional',
+    component: () => import('./pages/TheConditional.vue'),
+  },
   {
     path: '/multiple',
     name: 'Multiple',
-    component: () => import('./pages/multiple.vue'),
+    component: () => import('./pages/Multiple.vue'),
```

Comment:
Multiple.vue uses MultipleCanvas.vue, which doesn't exist anymore. MultipleCanvas was pretty useful in the past. It would be cool if we could keep it.

Reply:
You are right, I deleted it by mistake, let me add it again